.github/workflows/cmake-multi-platform.yml

```diff
@@ -60,96 +51,112 @@ jobs:
-      #
-      # To add more build types (Release, Debug, RelWithDebInfo, etc.) customize the build_type list.
       matrix:
         os: [ubuntu-latest, macos-latest]
         build_type: [Release]
         c_compiler: [clang]
-        # All [OS x compiler] items should be covered in include/exclude sections.
         include:
           - os: ubuntu-latest
-            c_compiler: clang
             cpp_compiler: clang++
-
           - os: macos-latest
-            c_compiler: clang
             cpp_compiler: clang++
 
     steps:
-    - uses: actions/checkout@v4
-
-    - name: Create Git LFS file list
-      run: git lfs ls-files -l |cut -d' ' -f1 |sort >.git/lfs-hashes.txt
-
-    - name: Restore Git LFS object cache
-      uses: actions/cache@v4
-      id: lfscache
-      with:
-        path: inflection/resources/org/unicode/inflection/dictionary
-        key: ${{ runner.os }}-lfsdata-v1-${{ hashFiles('.git/lfs-hashes.txt') }}
-        restore-keys: |
-          ${{ runner.os }}-lfsdata-v1-
-          ${{ runner.os }}-lfsdata
-
-    - name: Fetch any needed Git LFS objects and prune extraneous ones
-      if: steps.lfscache.outputs.cache-hit != 'true'
-      run: git lfs fetch --prune
-
-    - name: Check out Git LFS content
-      if: steps.lfscache.outputs.cache-hit != 'true'
-      run: git lfs checkout
-
-    - name: Check LFS restore the files or not after checkout
-      run: ls -l inflection/resources/org/unicode/inflection/dictionary/*
-
-    - name: Set reusable strings
-      # Turn repeated input strings (such as the build output directory) into step outputs. These step outputs can be used throughout the workflow file.
-      id: strings
-      shell: bash
-      run: |
-        echo "project-root-dir=${{ github.workspace }}/inflection" >> "$GITHUB_OUTPUT"
-        echo "build-output-dir=${{ github.workspace }}/inflection/build" >> "$GITHUB_OUTPUT"
-  
-    - uses: actions/cache/restore@v4
-      id: cache
-      with:
-        path: ~/icu
-        key: ${{ runner.os }}-icu-${ICU_MAJOR}-${ICU_MINOR}
-
-      # Install all the required dependencies for the macos
-    - name: Install ICU (Ubuntu/macos)
-      run: |
-        if [[ "${{ matrix.os }}" == "ubuntu-latest" ]]; then
-          echo "ICU_ROOT=~/icu/icu/usr/local" >> $GITHUB_ENV
-        elif [[ "${{ matrix.os }}" == "macos-latest" ]]; then
+      - uses: actions/checkout@v4
+
+      - name: Install Git LFS (Ubuntu/macOS)
+        run: |
+          if [[ "${{ matrix.os }}" == "ubuntu-latest" ]]; then
+            sudo apt-get update
+            sudo apt-get install -y git-lfs
+          elif [[ "${{ matrix.os }}" == "macos-latest" ]]; then
+            brew install git-lfs
+          fi
+          git lfs install
+
+      - name: Pull LFS files
+        run: git lfs pull
+        working-directory: ${{ github.workspace }}
+
+      - name: Debug Data Directory
+        run: ls -R inflection/resources/org/unicode/inflection/dictionary || true
```

Comment:
I think you meant `-o`, which shows the sizes, and dates, which are important. The `-R` lists the directory recursively.